Update vault.mdx (#2078)
hashicorp/terraform-website: content/enterprise/before-installing/data-storage-requirements/vault.mdx

  capabilities = ["read"]
}
```

### External Vault to Internal Vault Migration

> When migrating from an externally managed Vault service to the provided Terraform Enterprise internal Vault service, the backup process requires additional permissions to retreive keys from certain paths. To accomplish this the Vault policy needs to be modified to provided read access to paths. 

Reviewer: What do you think of this? Just wanting to make the text as clear and concise as possible! :-) 

```suggestion
The backup process requires additional permissions when migrating from an externally managed Vault service to the internal Terraform Enterprise Vault service. Modify the Vault policy to provide read access to the following paths. 
```
